### DISCUSSION TITLE
[Question]How to change the title and description that appear in Google search engine

### DISCUSSION THREAD

**Discussion by erde-ena15** [flet-dev/flet#3324] [Q&A] [answered]
### Question

I am hosting a flet app as a dynamic website. How can I change the title and description displayed in Google search engine?
![mosaic_20240520214136](https://github.com/flet-dev/flet/assets/61640670/510db154-4aac-4c36-a9d3-57b6933230df)

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/publish/web/dynamic-website#manifest
You need to modify the default `manifest.json` file.

**Reply by erde-ena15** [reply]
I have already added `/assets/manifest.json` but it is not reflected.
![スクリーンショット (47)](https://github.com/flet-dev/flet/assets/61640670/46bcd48c-0475-41b1-9d82-9b4b47effb0a)

I don't think search engine indexes will be updated unless crawlers visit my site again, so I tried posting a link to my site on Discord, but it still doesn't change.
![mosaic_20240521212409](https://github.com/flet-dev/flet/assets/61640670/093a6433-042a-4e2e-b741-0889fa0e5373)

**Reply by ndonkoHenri** [reply]
Ah, my bad, you mean the link preview!
When running the build command, make sure to provide the product name and description. 
(If you open `build/web/index.html` you will be able to see them)

Discord caches alot, so you might not be able to see the changes over there directly - no idea of how long it takes for a refresh over there.
I suggest you use the popular [opengraph](https://www.opengraph.xyz/).

**Reply by erde-ena15** [reply]
Thank you for letting me know about this useful site!
Sure, that method would work if you host it as a static website, but I'm hosting my flet app as a dynamic website, so I don't run the build command. Is there any other way?

**Comment by erde-ena15** [comment, marked as answer]
Adding `/assets/Manifest.json` seems to work when tested in the local environment, but the changes do not seem to be reflected on the server.
I couldn't find a good solution, so I took the strong way out by rewriting `flet-0.22.1/src/flet/web/index.html` and `manifest.json` in the flet library itself and creating a custom flet library. I would like to see an update in the future to make the change easier.
